server/src/main/java/io/crate/auth/AccessControlImpl.java

```diff
@@ -99,6 +103,9 @@
     private final UserLookup userLookup;
     private final SessionContext sessionContext;
 
+    private final static List<Privilege.Type> ALL_NON_AL_PRIVILEGES = List.of(Privilege.Type.DQL, Privilege.Type.DQL, Privilege.Type.DML);
```

Comment:
Looks like this isn't used?

Reply:
Yep, forgot to remove residuals of the old implementation, thanks.